Makefile.in

```diff
@@ -318,13 +318,20 @@ distprep: catman-do
 	$(AUTORECONF)
 	-rm -rf autom4te.cache
 
-install: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-sysconf host-key check-config
+install: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-iron-files install-sysconf host-key check-config
```

Comment:
Even if someone does a `make install`, should it install all the other stuff, or only `ironsftp`?